package com.tnsif.exceptionhandling;

public class Withexception {
	public static void main(String[] args) {
		int d=0;
		try {
			int a=4/0;
		}
		catch(ArithmeticException e) {
			System.out.println("exception caught "+e);
			System.out.println("hello");
		}
	}
}
